.github/actions/update_antora/action.yaml

```diff
@@ -0,0 +1,44 @@
+name: 'Update Antora docs'
+description: 'Updates the Antora docs'
+inputs:
+  version:  
+    description: 'Full version'
+    required: true
+  major-minor-version:  
+    description: 'Minor version'
+    required: true
+  patch-version:  
+    description: 'Patch version'
+    required: true
+  is-pre-release:  
+    description: 'If the release is a pre-release version'
+    required: true
+  release-branch:  
+    description: 'Release branch'
+    required: true
+  custom-facet-old-version:  
+    description: 'Version to replace in custom facet schema'
+    required: true
+  custom-facet-new-version:  
+    description: 'Version to update in custom facet schema'
+    required: true
+    
+runs:
+  using: "composite"
+  steps:
+    - name: Update
+      shell: bash
+      run: |
+        sed -i "s/^version: .*/version: ${{ inputs.patch-version }}/" docs/antora.yml
+        version=$( ${{ inputs.is-pre-release }} && echo ${{ inputs.version }} || echo ${{ inputs.major-minor-version }} )
+        sed -i "s/^display_version: .*/display_version: $version/" docs/antora.yml
+        sed -i "s/^prerelease: .*/prerelease: ${{ inputs.is-pre-release }}/" docs/antora.yml
+        sed -i "s/^    is-pre-release: .*/    is-pre-release: ${{ inputs.is-pre-release }}/" docs/antora.yml
+        find docs/modules/ROOT/attachments/ -type f -exec sed -i 's|${{ inputs.custom-facet-old-version }}|${{ inputs.custom-facet-new-version }}|g' {} +
+        git config --global user.email "aissemble@bah.com"
+        git config --global user.name "aiSSEMBLE Team"
+        git diff
+        git add .
+        git commit -S -m "Updating scripts and templates to ${{ inputs.version }}"
```

Comment:
A: I'd make the commit message more accurate. This is likely a carryover from when we updated more stuff manually instead of through the build process.

Reply:
Changed to "Updating Antora documentation to version ..."